spec/lib/weby/components/components_group_component_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+describe ComponentsGroupComponent do
+  it { should have_setting :html_class }
+  
+  it { should allow_value("class").for(:html_class) }
+  it { should_not allow_value("<body></bodyx>").for(:html_class) }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.